defaults/main/30_alternc-panel.yml

```diff
@@ -0,0 +1,214 @@
+# no: preserve existing TLDs, inserting new from playbook variables, updating those existing. 
+alternc_authorized_tld_ansible_exclusive: no
+
+# Preferred protocol for AlternC web panel. http or https
+# If you use altenrc-slavedns role, you would better put this variable in a group where are the master and the dns slave(s)
+alternc_desktop_protocol: https   
+
+# AlternC configuration variables (see web interface, Configuration panel -> Configure AltenrC variables )
+# (For full text `comment:`, we need to double escape the single quote ' (4 ', '''', to obtains one in the database) because they are parsed in an SQL sentence.)
+# (No way to make better using " around the whole text because the SQL sentence in this role has single quotes around the hash value.                           )
+## alternc_config_variables:
+## - name: archive_del_data
+##   value: 
+##   comment: 'If folder specified html folder of deleted user is archived, else it is deleted.'
+## - name: auth_ip_ftp_default_yes
+##   value: 1
+##   comment: 'This variable sets if you want to allow all IP address to access FTP by default. If the user start to define some IP or subnet in the allow list, only those he defined will be allowed.'
+## - name: debug_panel
+##   value: 1
+##   comment: 'Set it to 1 to enable panel debug in /var/log/alternc/bureau.log'
+## - name: default_certificate_fqdn
+##   value: '{{ alternc_debconf_desktopname }}'
+##   comment: 'FQDN of the certificate we will use as a default one before getting a proper one through any provider. If unsure, keep the default'
+## - name: default_dmarc_value
+##   value: 'p=none;pct=100;rua=%%ADMINMAIL%%;aspf=r;adkim=r'
+##   comment: 'This variable (if set) tells the DMARC/TXT DNS field that will be published as a DMARC entry, telling which policy you apply to this domain name. You can still change it later in the advanced DNS entries for ONE domain, this entry is only set once for new domains (or when you change it here). You can use %%ADMINMAIL%% or %%USERMAIL%% to substitute to admin-account or alternc user-account email address.'
+## - name: default_spf_value
+##   value: 'a mx ?all'
+##   comment: 'This variable (if set) tells the SPF/TXT DNS field that will be published as a SPF entry, telling which server(s) are allowed to send email for this one. You can still change it later in the advanced DNS entries for ONE domain, this entry is only set once for new domains (or when you change it here).'
+## - name: disk_quota_enable
+##   value: 1
+##   comment: 'Are disk quota enabled for this server'
+## - name: disk_quota_not_blocking
+##   value: 1
+##   comment: '0 - Block data when quota are exceeded (you need a working quota system) | 1 - Just show quota but don''''t block anything'
+## - name: favicon
+##   value: favicon.ico
+##   comment: 'You can specify a favicon, for example /images/my_logo.ico'
+## - name: force_https
+##   value: '{{ 1 if alternc_desktop_protocol == "https" else 0 }}'
```

Comment:
We are no longer setting this value... Maybe we should review again these variables and choose what to do in the role, by default or here, as part of the logic deriving from variables.